Sim/doc/FastSimulationUsingPapas.md

```diff
@@ -0,0 +1,137 @@
+Fast Simulation Using Papas
+==================================
+
+Contents
+
+-   [Overview](#overview)
+-   [Introduction](#introduction)
+-   [Installation Procedure](#installation-procedure)
+-   [What is Papas?](#what-is-papas?)
+-   [Running Papas in FCCSW](#running-papas-in-fccsw)
+-   [Planned Work](#planned-work)
```

Comment:
I'd rather have issues than this plan of work in the markdown doc. Maybe create issues and link to them? E.g. could prepend them all with "[papas]" and link to them like this (demonstrating with ACTS integration issues): https://github.com/HEP-FCC/FCCSW/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20%5BACTS-integration%5D